StarMapService/StarMapService.cs

```diff
@@ -193,6 +193,8 @@ public StarMapInfo getStarMapInfo(string systemName)
             }
             else
             {
+                /// Though not documented in the api, Anthor from EDSM has confirmed that this 
```

Comment:
First off: this is a prime example of a good code comment. 👍 
It conveys info that the source cannot, and cites a source -- in this case an unimpeachable one.
These are the sort of comments that I like.

Just an FYI, you only need `//` for comments. 

A `///` (or `/**`) is an identifier for systems like JavaDoc, docStrings, etc that attempt to build documentation from code comments.

You know, the sort of comments that describe a function's params and return type yet are nearly always oudated and wrong? Yes, those.

I have gone so far as to **ban** such things in my commercial jobs, because they lie so much that they cause far more harm than good.

Your best documentation is a suite of passing unit tests, because *they* CANNOT LIE.

o7